src/components/PathwayExercise/ExerciseContent/Assessment/index.js

```diff
@@ -0,0 +1,211 @@
+import React, { useState } from "react";
+import { Container, Box, Typography, Paper, Button } from "@mui/material";
+import useStyles from "./styles";
+import get from "lodash/get";
+import DOMPurify from "dompurify";
+
+const headingVarients = {};
+
+[Typography, "h2", "h3", "h4", "h5", "h6"].forEach(
+  (Name, index) =>
+    (headingVarients[index + 1] = (data) => (
+      <Name
+        className="heading"
+        dangerouslySetInnerHTML={{ __html: data }}
+        {...(index === 0 ? { component: "h1", variant: "h6" } : {})}
+      />
```

Comment:
There are a number of these components where you're dangerously setting the inner HTML.  As noted by its name, this property needs to be used very carefully when the user can have control over it in some way.  Since you want to try to avoid the possibility of forgetting to call `DOMPurify.sanitize` (as you may have here?) and since you may want to include other safeguards in the future for preventing against XSS attacks, it'd be a good idea to create some sort of component like the following:
```
function UnsafeHTML(props) {
  const {html, Container, ...otherProps} = props;
  const sanitizedHTML = DOMPurify.sanitize(html);
  return <Container {...otherProps} dangerouslySetInnerHTML={{ __html: sanitizedHTML}} /> 
}
```

So assuming you're keeping this as is (`<h_>` for everything except `<h1>`), you could then use this as follows:
```suggestion
      <UnsafeHTML
        Container={Name}
        className="heading"
        html={data}
        {...(index === 0 ? { component: "h1", variant: "h6" } : {})}
      />
```